integrations/llama_index/pyproject.toml

```
@@ -0,0 +1,65 @@
+[project]
+name = "ucai-llamaindex"
+version = "0.0.1"
+description = "Support for Unity Catalog functions as LlamaIndex tools"
+authors = [
+    { name="Ben Wilson", email="benjamin.wilson@databricks.com" }
+]
+readme = "README.md"
+license = { text="MIT" }
+requires-python = ">=3.9"
+dependencies = [
+    "llama-index>=0.11.0",
+    "unitycatalog-ai",
+]
+
+[project.optional-dependencies]
+dev = [
+  "pytest",
+  "typing_extensions",
+  # temporarily needed until unitycatalog-ai[databricks] is released
+  "databricks-sdk>=0.32.0",
+  "databricks-connect"
+]
+
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[tool.hatch.build]
+include = [
+  "src/ucai_llamaindex/*"
+]
+
+[tool.hatch.build.targets.wheel]
+packages = ["src/ucai_llamaindex"]
+
+[tool.ruff]
+line-length = 100
+target-version = "py39"
+
+[too.ruff.lint]
```

```suggestion
[tool.ruff.lint]
```
lol my typo bug found yesterday

---

ahhhh - found a few lint errors with this updated :D